backend/app/controllers/api/v1/investors_controller.rb

```diff
@@ -32,15 +33,22 @@ def show
       private
 
       def fetch_investor
-        return @investor = Investor.find(params[:id]) if fetching_by_uuid?
+        return @investor = Investor.includes(:account).find(params[:id]) if fetching_by_uuid?
```

Comment:
Not 100% sure, but when you use `find` to retrieve only one record, does it makes sense to use preloading with includes?

Reply:
sure, less of a problem than with list, but still one query instead of 2? and also makes you remember this other object is used for something in this call, maybe I like to be too explicit sometimes 😊

Reply:
Aha, oki douky. I though that 2 queries will be triggered as well in this case (one for loading investor and one for loading account) and power of includes is that when list of investors is obtained, it needs only one extra query to load all accounts. Whatever, will need to check doc at future to make sure :D

Reply:
Nice description of difference between includes/join/preload, etc. https://scoutapm.com/blog/activerecord-includes-vs-joins-vs-preload-vs-eager_load-when-and-where. From there:
```
Does includes always generate a separate query to fetch the records in the relationship?
No. includes will either use a separate query (like above) or a LEFT OUTER JOIN. If you have a where or order clause that references a relationship, a LEFT OUTER JOIN is used versus a separate query.
```
so in this case, it should truly use 2 queries 🤔

Reply:
OK, you're right, makes no sense! 👍